src/sections/Advice/index.js

```diff
@@ -1,11 +1,40 @@
 import { useEffect, useState } from "react"
+import AdviceSlip from "./components/AdviceSlip"
+import FavoriteSlipsList from "./components/FavouriteSlipsList"
 
 function AdviceSection() {
+  const [advice, setAdvice] = useState("")
+  const [favorites, setFavorites] = useState([])
+
+  const fetchAdvice = () => {
+    fetch("https://api.adviceslip.com/advice")
+    .then(response => response.json())
+    .then(data => setAdvice(data.slip.advice))
+  }
+
+  const addToFavorites = (advice) => {
+    if (!favorites.includes(advice)) {
+      setFavorites([ ...favorites, advice])
+    }
+  }
+
+  //extra for deleting the favorites list
+  const clearFavorites = () => {
+    setFavorites([])
+  }
+
+  useEffect(() => {
+    fetchAdvice()
+  }, [])
+
   return (
     <section>
       <h2>Advice Section</h2>
-      <section className="adivce-slip"></section>
-      <section className="favourtite-slips-list"></section>
+      <AdviceSlip advice={advice} getMoreAdvice={fetchAdvice} addToFavorites={addToFavorites} />
+      <FavoriteSlipsList favorites={favorites} />
+      {favorites.length > 0 && (
```

Comment:
`.length` is a truthy value, so you could remove the comparison and just do `favourites.length &&`

Reply:
Amazing, thank you for telling me! 
A small question, what if favourites.length would be 0? Then we put out a 0?

Reply:
If it's 0, it'll be treated as false. So by doing `{something.length && <Component />}`, the component will render if length is 1 or above, and won't render if it's 0

Reply:
Great, thank you!